lib/smart_answer_flows/coronavirus-employee-risk-assessment/questions/is_your_employer_asking_you_to_work.govspeak.erb

```diff
@@ -0,0 +1,14 @@
+<% render_content_for :pretext do %>
+  ##Your workplace should be closed
```

Comment:
Nitpick: 
```suggestion
  ## Your workplace should be closed
```

Reply:
for some reason - this adds space and doesn't align with the title, so I had to remove the space.